usage/conf/usage.conf

```diff
@@ -0,0 +1,28 @@
+env USER=media-service
+env USER_HOME=/home/media-service
+env JAR=/home/media-service/app.jar
+
+# Workaround for https://github.com/aws/aws-sdk-java/issues/123
+env JAVA_OPTS="-XX:MaxRAMFraction=2 -XX:InitialRAMFraction=2 -XX:+PrintGCDetails -XX:+PrintGCDateStamps -Xloggc:gc.log"
```

Comment:
I'd probably supersticiously set it, yeah. re memory, whichever is best I don't mind, but might be good to have consistency across the different run scripts, or else document why they differ?